### PR TITLE
jer: Remove implicit variables in ASN_DEBUGs

### PR DESCRIPTION
**Pull request by v0-e**
Removes nonexistent variables _hidden_ in ASN_DEBUG macros.

### CODE REVIEW

**Comment by mouse07410** [comment]
Hmm, I'm not sure. Perhaps, a better path would be finding a way to print those currently-hidden values? I disgust they could be quite useful in debugging.

**Comment by v0-e** [comment]
Indeed, that is a better option. I've replaced the offending variables.

File: skeletons/constr_CHOICE_jer.c [hunk 1]
```diff
@@ -249,12 +249,12 @@ CHOICE_decode_jer(const asn_codec_ctx_t *opt_codec_ctx,
...
 
         ASN_DEBUG("Unexpected JSON key [%c%c%c%c] in CHOICE [%s]"
-                  " (ph=%d, key=%s)",
+                  " (ph=%d)",
                   ch_size>0?((const uint8_t *)buf_ptr)[0]:'?',
                   ch_size>1?((const uint8_t *)buf_ptr)[1]:'?',
                   ch_size>2?((const uint8_t *)buf_ptr)[2]:'?',
                   ch_size>3?((const uint8_t *)buf_ptr)[3]:'?',
-                  td->name, ctx->phase, json_key);
+                  td->name, ctx->phase);
```
Comment (mouse07410) [inline skeletons/constr_CHOICE_jer.c:257]: If you find the key unexpected - wouldn't it make sense to print out what the offending key is?

File: skeletons/constr_SET_OF_jer.c [hunk 2]
```diff
@@ -79,8 +79,7 @@ SET_OF_decode_jer(const asn_codec_ctx_t *opt_codec_ctx,
             asn_dec_rval_t tmprval = {RC_OK, 0};
 
             /* Invoke the inner type decoder, m.b. multiple times */
-            ASN_DEBUG("JER/SET OF element [%s]", elm_tag(*element->name) ? 
-                      element->name : element->type->xml_tag);
```
Comment (mouse07410) [inline skeletons/constr_SET_OF_jer.c:83]: I'd prefer to see the element name here - either its value, or printed `"NULL"` if empty.

**Review by mouse07410 — CHANGES_REQUESTED** [review]

File: skeletons/constr_CHOICE_jer.c (hunk 1 above)
Comment (v0-e) [reply skeletons/constr_CHOICE_jer.c:257]: The wording is wrong, should be a general set of JSON characters, or "token", which is not handled by the previous logic. It is not just a "key".

File: skeletons/constr_SET_OF_jer.c (hunk 2 above)
Comment (v0-e) [reply skeletons/constr_SET_OF_jer.c:83]: I'll leave it to print the element name if exists, or else its type.

**Review by v0-e — COMMENTED** [review]
Fixed requested changes.

File: skeletons/constr_CHOICE_jer.c [hunk 3]
```diff
@@ -140,12 +140,12 @@ CHOICE_decode_jer(const asn_codec_ctx_t *opt_codec_ctx,
...
 
         scv = jer_check_sym(buf_ptr, ch_size, NULL);
-        ASN_DEBUG("JER/CHOICE checked [%c%c%c%c] vs [%s], scv=%d",
+        ASN_DEBUG("JER/CHOICE checked [%c%c%c%c] of [%s], scv=%d",
                   ch_size>0?((const uint8_t *)buf_ptr)[0]:'?',
                   ch_size>1?((const uint8_t *)buf_ptr)[1]:'?',
                   ch_size>2?((const uint8_t *)buf_ptr)[2]:'?',
                   ch_size>3?((const uint8_t *)buf_ptr)[3]:'?',
-                  json_key, scv);
+                  td->name, scv);
```
Comment (mouse07410) [inline skeletons/constr_CHOICE_jer.c:148]: Thanks!

**Review by mouse07410 — CHANGES_REQUESTED** [review]

**Pull request merged**